datafusion/sql/tests/cases/diagnostic.rs

```diff
@@ -136,7 +137,7 @@ fn test_table_not_found() -> Result<()> {
     let query = "SELECT * FROM /*a*/personx/*a*/";
     let spans = get_spans(query);
     let diag = do_query(query);
-    assert_eq!(diag.message, "table 'personx' not found");
+    assert_snapshot!(diag.message, @"table 'personx' not found");
```

Comment:
This is fine though single string comparisons are probably fine to just leave as assert_eq in the future